### PR TITLE
update nina tile's css

### PR DESCRIPTION
**Pull request by astux7**
Update nina with new style

### CODE REVIEW

File: src/tile-nina/tile-nina.scss [hunk 1]
```diff
@@ -34,32 +24,57 @@
...
 
   &.small {
-    .tile-title,
-    .tile-details {
-      font-size: desktop-font(26);
+    .tile-title {
+      font-size: desktop-font(36);
       line-height: 1.2em;
+      color: $white;
+      padding-top: 1em;
```
Comment (benjy2429) [inline src/tile-nina/tile-nina.scss]: Being a bit picky but could we reduce this to 0.5em instead? There's a lot of padding at the top of the tile compared to the bottom:

screen shot 2016-08-26 at 8 42 47 am

**Comment by benjy2429** [comment]
:shipit:

For reference, the new tile design looks like this:

screen shot 2016-08-26 at 9 50 41 am

File: src/tile-nina/tile-nina.scss [hunk 2]
```diff
@@ -11,21 +12,10 @@
...
-    img {
-      width: 100%;
-    }
-  }
-
   .tile-content {
-    margin-top: 49.25%;
-    margin-bottom: 7%;
+    padding-bottom: 99.99%;
+    @extend .polaris-bg-sky-1;
```
Comment (joe-bell) [inline src/tile-nina/tile-nina.scss:17]: You're using quite a lot of polaris-globals in this CSS - which we are planning to deprecate.
[Some of which has already been deprecated](https://github.com/skyglobal/polaris-global/blob/master/globals/_colours.scss).

Can we amend this to use the backgrounds and colors in Toolkit instead?

There's some more info here -  http://sky-uk.github.io/toolkit/#core-gradients

File: src/tile-nina/tile-nina.scss [hunk 3]
```diff
@@ -11,21 +12,10 @@
...
+    color: $white;
```
Comment (joe-bell) [inline src/tile-nina/tile-nina.scss]: See above - this should be `color: color(white);`

File: src/tile-nina/tile-nina.scss [hunk 4]
```diff
@@ -34,32 +24,57 @@
 
   .tile-nina-button {
     @include font($skytext-med);
-    margin: 10% auto 0;
+    margin: 2.5% auto 0;
     display: inline-block;
-    padding: .5em 1em;
-    border: 2px solid $sky-blue;
+    padding: .3em 1.3em;
+    border: 1px solid $white;
```
Comment (joe-bell) [inline src/tile-nina/tile-nina.scss]: Same as above, this should be `border: 1px solid color(white);`

File: src/tile-nina/tile-nina.scss [hunk 5]
```diff
@@ -34,32 +24,57 @@
...
     border-radius: 4px;
-    color: $sky-blue;
-    font-size: 20px;
+    color: $white;
```
Comment (joe-bell) [inline src/tile-nina/tile-nina.scss]: Same as above, this should be `color: color(white);`

File: src/tile-nina/tile-nina.scss [hunk 6]
```diff
@@ -34,32 +24,57 @@
...
+    @include mobile {
+      font-size: desktop-font(84);
+    }
   }
 
   &:hover .tile-nina-button {
-    color: $white;
-    background-color: $sky-blue;
+    background-color: $white;
+    color: $sky-blue;
```
Comment (joe-bell) [inline src/tile-nina/tile-nina.scss:46]: Same as above, this should be 

```
background-color: color(white);
color: color(brand);
```

File: src/tile-nina/tile-nina.scss [hunk 7]
```diff
@@ -34,32 +24,57 @@
...
   }
 
   &.small {
-    .tile-title,
-    .tile-details {
-      font-size: desktop-font(26);
+    .tile-title {
+      font-size: desktop-font(36);
       line-height: 1.2em;
+      color: $white;
```
Comment (joe-bell) [inline src/tile-nina/tile-nina.scss]: Same as above, this should be `color: color(white);`

File: src/tile-nina/tile-nina.scss [hunk 8]
```diff
@@ -34,32 +24,57 @@
...
+      @include mobile {
+        font-size: desktop-font(124);
+      }
+    }
+
+    .tile-subtitle {
+      padding-top: 1.2em;
+      font-size: desktop-font(20);
+      line-height: 1.5em;
+      color: $white;
```
Comment (joe-bell) [inline src/tile-nina/tile-nina.scss]: Same as above, this should be `color: color(white);`

File: src/tile-nina/tile-nina.scss [hunk 9]
```diff
@@ -34,32 +24,57 @@
 
   .tile-nina-button {
     @include font($skytext-med);
```
Comment (joe-bell) [inline src/tile-nina/tile-nina.scss]: Remove and instead use `font-weight: bold;` 😃

**Comment by benjy2429** [comment]
@astux7 @joebell93 
From the Sky Pages meeting today, there is a steer to deprecate the skyglobal/tiles repo and move all the CSS into the toolkit. This work is currently ongoing but once finished, the tiles repo will no longer be used.

Our best approach for the Nina tile is to copy the new CSS rules directly into Sky Pages. This will prevent any conflicts for the Toolkit guys and will also prevent the new styles being lost when the repo is deprecated.

Closing this PR and moving the styles into Sky Pages instead.

**Pull request closed without merging**